Cinema system/Cinema system/Controllers/CinemasController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using CinemaSystem.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace CinemaSystem.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CinemasController : ControllerBase
+    {
+
+        private CinemaDBContext _db;
+
+        public CinemasController(CinemaDBContext context)
+        {
+            _db = context;
+        }
+
+        [HttpGet]
+        public IEnumerable<Cinema> Get()
+        {
+            IEnumerable<Cinema> cinemas = _db.Cinemas.AsEnumerable();   
```

Comment:
What does AsEnumerable actually do? Does it required here? Why not using var here?